frontend/src/components/auth/Signup/AccountDetails.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  Box,
+  Button,
+  Container,
+  FormControl,
+  IconButton,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import React, { useContext, useReducer, useState } from "react";
+import { Redirect, useHistory } from "react-router-dom";
+
+import authAPIClient from "../../../APIClients/AuthAPIClient";
+import {
+  HOME_PAGE,
+  LANDING_PAGE,
+  VERIFICATION_PAGE,
+} from "../../../constants/Routes";
+import AuthContext from "../../../contexts/AuthContext";
+import { AuthenticatedUser } from "../../../types/AuthTypes";
+import Header from "../../common/Header";
+import { BackArrow, CloseIcon } from "../../common/icons";
+import { initialState, passwordVerificationReducer } from "../reducer";
+import {
+  checkForLowerCase,
+  checkForNumbers,
+  checkForSpecialCharacters,
+  checkForUpperCase,
+  checkLength,
+} from "../utilities";
+import { MandatoryInputDescription, PasswordRequirement } from "./components";
+
+const AccountDetails = ({
+  formValues,
+  setForm,
+  navigation,
+}: {
+  navigation: any;
```

Comment:
instead of any, can we change it to the types like you have it for CreateAccount component?